test/e2e/package.json

```diff
@@ -49,7 +49,7 @@
 		"mocha-steps": "^1.1.0",
 		"node-slack-upload": "1.2.1",
 		"png-itxt": "1.3.0",
-		"prettier": "npm:wp-prettier@1.19.1",
+		"prettier": "npm:wp-prettier@2.0.5-alpha-2",
```

Comment:
Unrelated — just wanted to flag here so that we don't accidentally get it in here.

Reply:
I've removed this change during rebase.